src/App.css

```diff
@@ -1 +1,4 @@
-
+* {
+    box-sizing: border-box;
+    overflow: hidden;
+}
```

Comment:
Can you add the windows xp font = Tahoma

Reply:
Added